### PR TITLE
Linux introducction

### PR DESCRIPTION
**Pull request by wozgeass**
Hi, I want to contribute with you repo and I'm going to write the introduction and history about the linux kernel.

### CODE REVIEW

**Comment by 0xAX** [comment]
Hi @wozgeass, this is great idea! Let me know that you will be ready and I will review and apply it.

Thank you

File: Introduction/Introduction.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
```
Comment (lstor) [inline Introduction/Introduction.md:4]: Begi**n**ning

File: Introduction/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
```
Comment (lstor) [inline Introduction/Introduction.md:6]: Not quite sure what you're trying to say here. Perhaps: Linux' importance as an operating system has grown in recent years, and many people use it in their daily lives.

File: Introduction/Introduction.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+But nobody like remember the history about the Linux kernel.
+
+It s now installed on millions of systems and used by home users and professionals alike for a wide range of tasks. From miniature embedded systems in wristwatches to massively parallel mainframes, the are countless way of exploiting linux productively.
```
Comment (lstor) [inline Introduction/Introduction.md:9]: - It **i**s
- wrist watches (with space)
- the**re** are
- perhaps another word than 'exploiting' is more suitable. For example 'utilizing'?
- **L**inux (capitalized)

File: Introduction/Introduction.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
+But nobody like remember the history about the Linux kernel.
```
Comment (lstor) [inline Introduction/Introduction.md:7]: - Why is the 'like' there? Unless it conveys something I'm not picking up, I think it's better without it.
- nobody remember**s**

File: Introduction/Introduction.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+It s now installed on millions of systems and used by home users and professionals alike for a wide range of tasks. From miniature embedded systems in wristwatches to massively parallel mainframes, the are countless way of exploiting linux productively.
+
+This the first part to undestand if you want to begin with this topics.
```
Comment (lstor) [inline Introduction/Introduction.md:11]: - This **is**
- unde**r**stand
- topic (not topic<s>s</s>)

File: Introduction/Introduction.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+Linux looks like any other Unix systems; indeed, UNIX compatiblity has a major design and goal of the Linux Kernel project and it is considered as the core of the Operating Systems.
```
Comment (lstor) [inline Introduction/Introduction.md:12]: - any other system (not system<s>s</s>)
- I'd consider dropping the semicolon
- compatib**i**lity
- has **been** a major
- I think I know what you mean, but I would rather write 'design goal' than 'design and goal'.
- What do you mean by 'it is considered as the core of the OSes'?

File: Introduction/Introduction.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+The First difference betwen any other programs in the system consist that controls the hardware and the applications but the applications do not communicate with the hardware directly, instead the go to the kernel. In summary
```
Comment (lstor) [inline Introduction/Introduction.md:13]: - **f**irst (not capitalized)
- betw**e**en
- The sentence is hard to understand. What exactly do you mean here?

File: Introduction/Introduction.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+In 1991 a Finish student named of Linus Benedict Torvalds begin with his development. He released the first linux version on September 1991, and on february 1992, he lisensed the kernel under the GPL licence.
```
Comment (lstor) [inline Introduction/Introduction.md:15]: - Fi**n**nish
- named <s>of</s> Linus (not **of**)
- beg**a**
- **L**inux (capitalized)
- **In** September. (Also, it was September 25 if memory serves me correctly.)
- Period after 1991 -- start a new sentence.
- li**c**ensed

File: Introduction/Introduction.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+![image](https://lh4.googleusercontent.com/-Y_ESIDYRHpk/AAAAAAAAAAI/AAAAAAAAQtI/GsbYt_CjQmI/photo.jpg)
+
+
+
+--------------------------------------------------------------------------------
+
+* [Linus Torvalds](http://es.wikipedia.org/wiki/Linus_Torvalds)
```
Comment (lstor) [inline Introduction/Introduction.md:23]: **en**.wikipedia.org, not Spanish wikipedia :-)

File: Introduction/Kernel_in_Time.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
+History of Linux Kernel.
+================================================================================
+In the previous post [Introduction]() we see that the linux kernel began in
```
Comment (lstor) [inline Introduction/Kernel_in_Time.md:3]: Unfinished sentence?

**Comment by lstor** [comment]
I came across this while bored, and decided to do a quick check on grammar. Hope that's okay :smiley:

**Comment by wozgeass** [comment]
There's no, problem for your check on grammar thanks for that.
And excuse me for have not ended this part quickly, but i have been working without a break.
May be  the next week i have some time for a relax to continue but I will add your changes.

**Comment by 0xAX** [comment]
Hello @wozgeass, any progress here?

**Comment by 0xAX** [comment]
I will close this issue as there is no progress, please feel free to reopen it if somebody will have to say something

**Pull request closed without merging**